src/controller/AbstractDnssdDiscoveryController.cpp

```diff
@@ -25,6 +25,32 @@
 namespace chip {
 namespace Controller {
 
+bool compareIpAddresses(const size_t sourceNumIPs, const size_t destinationNumIPs, const Inet::IPAddress * source,
```

Comment:
Why isn't this taking two `const Span<Inet::IPAddress> &`?

Reply:
Inet::IPAddress is not pointer in OnNodeDiscovered.
When implemented in Span format, a Span variable must be created in OnNodeDiscovered, so it was implemented like this.

Reply:
I modified using "Span" class.